exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -338,3 +338,127 @@ drill.exec: {
 drill.jdbc: {
   batch_queue_throttling_threshold: 100
 }
+
+drill.exec.options:  {
```

Comment:
Also, any chance we can alphabetize the list? Just run it through the `sort` command? Will make future human access just a bit easier...